e2e_tests/.env.example

```diff
@@ -3,22 +3,29 @@ VALID_USER=
 VALID_PASSWORD=
 
 # Address to run the test against
-HOME_URL=
+HOME_URL=https://app.deriv.com/
 
 # This is used to bypass duo page.
 QABOX_DUO_PASSWORD=
 QABOX_DUO_EMAIL=
 QABOX_DUO_CODE=
 
 # The URL APP ID that is used to connect to the backend
-QABOX_SERVER=
-QABOX_APP_ID=
+QABOX_SERVER=www.binaryqa10.com
+QABOX_APP_ID=1013
 
 # Used to tell tests to connect to QA
-QA_SETUP=false
+QA_SETUP=0
 
 QAW_ARTIFACT_PATH=/home/mamali/qawolf
```

Comment:
Just a question: Is this also used in playwright or we can remove this?

Reply:
Thanks for pointing it out. it should've been E2E artifacts path.